Client/Ecosia/MMP/MMP.swift

```diff
@@ -6,6 +6,8 @@ import Foundation
 import Core
 import Shared
 
+// Test workflow trigger
```

Comment:
I'm guessing all those comments are meant only for triggering and should be removed before merging, just adding this comment as a reminder.

Reply:
Correct 😉 - added in ## Other section